### PR TITLE
exclude all sub-pages of user-pages from "new community members"

### PR DESCRIPTION
**Pull request by sknebel**
Fix for #4 – this seemed easy enough even for my nearly non-existent level of PHP skills ;)

### CODE REVIEW

File: generate-wiki-summary.php [hunk 1]
```diff
@@ -74,7 +74,8 @@
 
 		// Summary
 		if($page['type'] == 'new') {
-			if(preg_match('/^user:/i', $title) && !preg_match('/\.css$/i', $title))
+			//subpages of user-pages are excluded (e.g. user css or personal test pages)
```
Comment (strugee) [inline generate-wiki-summary.php]: Probably want `// Subpages` instead of `//subpages`, for consistency with other comments

Comment (strugee) [reply generate-wiki-summary.php]: Also CSS is all-caps

File: generate-wiki-summary.php [hunk 2]
```diff
@@ -74,7 +74,8 @@
 
 		// Summary
 		if($page['type'] == 'new') {
-			if(preg_match('/^user:/i', $title) && !preg_match('/\.css$/i', $title))
```
Comment (strugee) [inline generate-wiki-summary.php:77]: The regex for `.css` seems to have disappeared here; what happened to it?

**Review by strugee — CHANGES_REQUESTED** [review]
@sknebel I'm not a committer for this repo but I saw your PR pop up in my notifications and thought I'd take a look - hope you don't mind :)

You might also want to adjust your commit message so it's [capitalized](https://chris.beams.io/posts/git-commit/#capitalize) and so the body includes "Fixes #4" (so issue #4 autocloses).

File: generate-wiki-summary.php (hunk 2 above)
Comment (sknebel) [reply generate-wiki-summary.php:77]: custom CSS is just a specific type of subpage

**Pull request merged**